.jcheck/conf

```diff
@@ -29,6 +29,7 @@ version=1.0
 
 [checks]
 error=author,reviewers,whitespace
+warning=copyright
```

Comment:
I am not sure if we should make it an error or warning. Maybe warning first and later we can change it to error